### PR TITLE
Pull request ( 29/01/2021)

### PR DESCRIPTION
**Pull request by AadarshGhodasara**
**Today's task**

- Create a home Module.
- some changes in AddClockphoto.js and AddFramePhoto.js file.
- Done proper working flow for admin and normal user.

### CODE REVIEW

File: src/Pages/addPhoto/AddFramePhoto.js [hunk 1]
```diff
@@ -87,50 +88,50 @@ class AddPhoto extends React.Component {
...
-                                </Button>
-                               <Button 
-                                    className='uploadFrame-btn' 
-                                    variant="contained" 
-                                    color="secondary"
-                                    startIcon={<DeleteIcon />}
-                                >
-                                    <label className='btn-text-css' htmlFor='framedel' >Delete</label>
-                                </Button>
+            <div>
```
Comment (simformniki) [inline src/Pages/addPhoto/AddFramePhoto.js]: You can use **React Fragment** to wrap multiple components if you are not assigning any **className** to it.

File: src/Pages/addPhoto/AddClockPhoto.js [hunk 2]
```diff
@@ -86,54 +88,57 @@ class AddPhoto extends React.Component {
...
-                                </Button>
-                               <Button 
-                                    className='upload-btn' 
-                                    variant="contained" 
-                                    color="secondary"
-                                    startIcon={<DeleteIcon />}
-                                >
-                                    <label className='btn-text-css' htmlFor='del' >Delete</label>
-                                </Button>
+            <div>
```
Comment (simformniki) [inline src/Pages/addPhoto/AddClockPhoto.js]: You can use **React Fragment** to wrap multiple components if you are not assigning any **className** to it.

File: src/Components/checkUserStatus/UserStatus.js [hunk 3]
```diff
@@ -4,6 +4,10 @@ import history from '../History';
 import LoaderModal from '../Loader/LoaderModal';
 // import Contact from '../../Pages/Content/content';
 import './CheckUserStatus.css';
+export let adminVerify=false;
```
Comment (simformniki) [inline src/Components/checkUserStatus/UserStatus.js]: let's discuss this over a call on Monday.

File: src/App.js [hunk 4]
```diff
@@ -1,84 +1,37 @@
...
-// import reducer from './reducer/reducer';
-import CheckUserStatus from './Components/checkUserStatus/UserStatus';
-// import AddPhoto from './Pages/addPhoto/AddPhoto';
-// import ShowImages from './Pages/showImages/showImages';
-// import TempLogin from './Pages/Login/TempLogin';
-// const  store = createStore(reducer);
-
-// console.log(store.getState());
+import AddClockPhoto from './Pages/AddPhoto/AddClockPhoto';
+import AddFramePhoto from './Pages/AddPhoto/AddFramePhoto';
```
Comment (simformniki) [inline src/App.js]: You can use **lazy loading** to import components for the routes.

**Review by nikip28 — APPROVED** [review]

**Pull request merged**